### ISSUE TITLE
Imported NPC statblocks have mysterious -5 to-hit

### ISSUE THREAD

**Issue by Zharthalus** [kypvalanx/Foundry-VTT-StarWars-SagaEdition#510]
Imported some Stormtroopers for my campaign and noticed that they are attacking with a -5 penalty at all times, at first I thought it was autofire, but screenshots attached below prove that it happens even with autofire turned off.

![firefox 12_10_2024-03_11_10](https://github.com/user-attachments/assets/80b8c551-cf19-456f-a648-dbea87afcf1e)

![firefox 12_10_2024-03_11_32](https://github.com/user-attachments/assets/85e138cd-d6d8-46fa-8080-80d81089307d)

I attempted to delete the blaster rifle and give it back to the sheet, I also attempted to delete and re-add the weapon proficiency (rifles) feat to the sheet. Neither did anything.

If this is not a bug, then I would be curious how to resolve it so I do not have to manually add +5 to every attack roll.

**Comment by LeKopic** [comment]
1) What version are you on?
2) Does the main actor have autofire enabled in his character sheet?

**Comment by Zharthalus** [comment]
> 1. What version are you on?
> 2. Does the main actor have autofire enabled in his character sheet?

I am on the latest version: 12.3.2

And no, the second image attached has two -5 penalties, one of those is from autofire being turned on. The first image is me turning autofire off, both in the weapon settings and in the modes tab. 

I toyed around with trying to fix this manually and the only solution I found is that enabling and then disabling the 'strip autofire' modification works for removing it, I am not an expert on this sheet in the slightest, but I can only presume that it was simply applying the autofire penalty twice. While this is technically _a_ solution to the problem, every further import still has this happening, to provide some evidence, here is a freshly imported CL1 Stormtrooper NPC sheet with this problem.

![firefox 12_10_2024-14_11_24](https://github.com/user-attachments/assets/2a7d38d3-37a9-43b5-a633-2498000aa786)
![firefox 12_10_2024-14_13_25](https://github.com/user-attachments/assets/b11618b0-7d42-468c-9da5-745043670804)

No clue if this is just happening to me, but it is still rather strange.
